db/migrate/20180928172525_generate_web_channel_url_key.rb

```diff
@@ -0,0 +1,17 @@
+class GenerateWebChannelUrlKey < ActiveRecord::Migration[5.1]
+  class Channel < ActiveRecord::Base
+  end
+
+  def up
+    Channel.reset_column_information
+
+    Bot.where.not(uuid: nil).find_each do |bot|
```

Comment:
You should define the bot class here in the migration too, and link it with the channel using `belongs_to` and `has_many`.